hadoop-ozone/dist/src/main/compose/test-all.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )
+ALL_RESULT_DIR="$SCRIPT_DIR/result"
+
+mkdir -p "$ALL_RESULT_DIR"
+rm "$ALL_RESULT_DIR/*"
+
+RESULT=0
+IFS=$'\n'
+# shellcheck disable=SC2044
```

Comment:
shellcheck:20: note: Double quote to prevent globbing and word splitting. [SC2086]